### PR TITLE
Add train script and sample for SSD

### PR DESCRIPTION
**Pull request by knorth55**
You can train SSD with 73B2 kitchen dataset and try sample launch.
I trained the model with my machine and it only takes 10 minutes for 100 epoch :).

cc. @Affonso-Gui

### CODE REVIEW

File: jsk_perception/node_scripts/ssd_object_detector.py [hunk 1]
```diff
@@ -26,6 +26,9 @@
 from chainercv.visualizations import vis_bbox
 
 
+chainer.config.cv_resize_backend = 'cv2'
```
Comment (YutoUchimi) [inline jsk_perception/node_scripts/ssd_object_detector.py]: Could you also add this line to `train_ssd.py`?

**Comment by Affonso-Gui** [comment]
@knorth55 Looking good! Will test it on the real robot next week

File: doc/jsk_perception/training_scripts/train_ssd.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  Number of images used simultaneously in each iteration.
+
+  You should decrease this number when you face memory allocation error.
+
+- `--max-epoch` (`int`, default: `100`)
+
+  Stop trigger for training.
+
+- `--out_dir` (`string`, default: `${ROS_HOME}/learning_logs/<timestamp>`)
```
Comment (YutoUchimi) [inline doc/jsk_perception/training_scripts/train_ssd.md]: Thank you for updating! Just one more thing: `--out_dir` -> `--out-dir`.

File: jsk_perception/scripts/train_ssd.py [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    args = parser.parse_args()
+
+    train_dataset = DetectionDataset(args.train_dataset_dir)
+    fg_label_names = train_dataset.fg_class_names
+
+    if args.model == 'ssd300':
+        model = SSD300(
+            n_fg_class=len(fg_label_names),
+            pretrained_model='imagenet')
+    elif args.model == 'ssd512':
```
Comment (Affonso-Gui) [inline jsk_perception/scripts/train_ssd.py]: `args.model_name`, or maybe set a `dest` in the argument?

Comment (knorth55) [reply jsk_perception/scripts/train_ssd.py]: Thx, I updated

Comment (knorth55) [reply jsk_perception/scripts/train_ssd.py]: thx, i updated

**Pull request merged**